webservice/template/admin/course/index.tmpl

```diff
@@ -1,24 +1,49 @@
 {{define "title"}}Courses{{end}}
-{{define "head"}}{{end}}
+{{define "head"}}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
jquery and bootstrap should be in the "foot", and are in the base.tmpl

Reply:
If I move both or either onw of theese to `foot`, the tooltip won't work properly